new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -1074,6 +1074,7 @@ private static void collectAppAvaiability(
       
       if (count == 0) {
         logger.info("XXXXXXXXXXX: application not available XXXXXXXX");
```

Comment:
What are the "XXX" and "YYY"? What application is it?

Reply:
Those "XXX" and "YYY" marks just make it easier for people to notice the failure/healthy conditions. The failure case "XXX" is INFO, and "YYY" is "FINE".